### DISCUSSION TITLE
WindowsがKeyballを認識しなくなってしまった

### DISCUSSION THREAD

**Discussion by haru086** [Yowkees/keyball#457] [Q&A]
ProMicroのUSBをマグネット端子で何度か抜き差ししているうちに、WindowsがKeyball39を認識できなくなってしまいました。
OLEDにKeyball39のロゴも表示されず、砂粒みたいなランダムな細かいドットが表示された状態になります。
しかし不思議なことにUSBケーブルを変えたり接続ポートを変更すると、たまに認識できるようになりますが、気が付いたらまた認識しなくなってしまいます。
リセットボタンを素早く2回押してSWを再D/Lしても直りませんでした。
対処法を教えて頂けないでしょうか。
![スクリーンショット 2023-12-26 133511](https://github.com/Yowkees/keyball/assets/154874354/29bde3f3-45c5-4f9a-9ab1-c18b7bc8bc8c)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

症状から推測すると、USB接続が不安定でProMicroがうまく起動できていないのではないでしょうか。
USBケーブルを別のもの（信頼性の高そうなもの）に変更して様子をみていただけますでしょうか。

以上、お手数ですがご対応お願いいたします。

**Reply by haru086** [reply]
早速のご回答ありがとうございます。

色々と試した結果、通信が不安定なUSBポート（モニターから接続できるポート）と、通信に不向きなUSBケーブルがあることが分かりました。
アドバイスありがとうございます。
USBケーブルと接続ポートを見直すことで、再びKeyballが使えるようになりました！

**Comment by Yowkees** [comment]
いろいろ工夫していただき、無事に動作するようになったようで良かったです。安心しました。
ぜひ今後ともご愛用いただければ幸いです。
